src/core/net-nonblock.c

```diff
@@ -27,13 +27,11 @@
 
 /* nonblocking gethostbyname(), ip (IPADDR) + error (int, 0 = not error) is
    written to pipe when found PID of the resolver child is returned */
-int net_gethostbyname_nonblock(const char *addr, GIOChannel *pipe,
-			       int reverse_lookup)
+int net_gethostbyname_nonblock(const char *addr, GIOChannel *pipe)
```

Comment:
This looks API breaking, keep the param

Reply:
maybe better to break the API since the feature is gone?

Reply:
I don't think this feature is as important as the ability to do DNS resolutions at all. Callers won't really care if reverse lookup is done or not.

Reply:
Good catch, thank you! It's there again.